### PR TITLE
chore(manifest): populate static site manifest

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/template/templates/workloads/services/static-site/manifest.yml [hunk 1]
```diff
@@ -16,19 +16,9 @@ files:
...
-    {{- end}}
-    {{- if $.Recursive}}
-    recursive: {{$.Recursive}}
-    {{- end}}
-    {{- if $.Reinclude.StringSlice}}
-    reinclude: {{$.Reinclude.StringSlice}}
-    {{- end}}
-    {{- if $.Exclude.StringSlice}}
-    exclude: {{$.Exclude.StringSlice}}
+  - source: {{$file.Source}}
```
Comment (iamhopaul123) [inline internal/pkg/template/templates/workloads/services/static-site/manifest.yml:18]: hmm do we not write destination?

Comment (huanjani) [reply internal/pkg/template/templates/workloads/services/static-site/manifest.yml:18]: added to be the same as the case without specified sources. wondering what our manifest validation should look like for `destination`....

File: internal/pkg/deploy/cloudformation/stack/static_site_test.go [hunk 2]
```diff
@@ -247,7 +247,10 @@ func TestStaticSite_Parameters(t *testing.T) {
 	}
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-			testManifest := manifest.NewStaticSite("frontend")
+			testManifest := manifest.NewStaticSite(manifest.StaticSiteProps{
+				Name:             "frontend",
+				StaticSiteConfig: manifest.StaticSiteConfig{},
```
Comment (dannyrandall) [inline internal/pkg/deploy/cloudformation/stack/static_site_test.go]:
```suggestion
```
nit: it's the zero value already 😊

File: internal/pkg/template/templates/workloads/services/static-site/manifest.yml [hunk 3]
```diff
@@ -16,19 +16,10 @@ files:
...
+    {{- if $file.Recursive}}
+    destination: assets
```
Comment (dannyrandall) [inline internal/pkg/template/templates/workloads/services/static-site/manifest.yml]: why `assets`..? isn't the bucket an asset bucket already?

**Review by dannyrandall — COMMENTED** [review]
looks pretty good, just a question!

File: internal/pkg/template/templates/workloads/services/static-site/manifest.yml (hunk 3 above)
Comment (huanjani) [reply internal/pkg/template/templates/workloads/services/static-site/manifest.yml]: changed per offline conversation

**Review by dannyrandall — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**